### PR TITLE
Update docformatter to latest version

### PR DESCRIPTION
**Pull request by tanertopal**
### Description

This PR does two things:

1. Use same `docformatter` version in all projects
2. Update all to the latest version

### Related issues/PRs

* Blocked by: #2956

### Checklist

- [x] Implement proposed change
- [ ] Write tests
- [ ] Update [documentation](https://flower.dev/docs/writing-documentation.html)
- [ ] Update the changelog entry below
- [x] Make CI checks pass
- [x] Ping maintainers on [Slack](https://flower.dev/join-slack/) (channel `#contributions`)

### CODE REVIEW

File: baselines/fedavgm/pyproject.toml [hunk 1]
```diff
@@ -48,14 +48,15 @@ matplotlib = "^3.7.2"
...
-docformatter = "==1.5.1"
+docformatter = "==1.7.5"
 mypy = "==1.4.1"
 pylint = "==2.8.2"
 flake8 = "==3.9.2"
 pytest = "==6.2.4"
 pytest-watch = "==4.2.0"
 ruff = "==0.0.272"
 types-requests = "==2.27.7"
+virtualenv = "==20.21.0"
```
Comment (tanertopal) [inline baselines/fedavgm/pyproject.toml:59]:
```suggestion
```

File: baselines/fedmlb/pyproject.toml [hunk 2]
```diff
@@ -47,14 +47,15 @@ psutil = "5.9.5" # this is just to log some info about memory usage
...
-docformatter = "==1.5.1"
+docformatter = "==1.7.5"
 mypy = "==1.4.1"
 pylint = "==2.8.2"
 flake8 = "==3.9.2"
 pytest = "==6.2.4"
 pytest-watch = "==4.2.0"
 ruff = "==0.0.272"
 types-requests = "==2.27.7"
+virtualenv = "==20.21.0"
```
Comment (tanertopal) [inline baselines/fedmlb/pyproject.toml:58]:
```suggestion
```

File: baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist/model.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def train(
     verbose: bool = False,
 ) -> Tuple[float, float, Optional[float], Optional[float]]:
     """Train a given model with CrossEntropy and SGD (or some version of it
-    like batch-SGD).
+    like batch- SGD).
```
Comment (danieljanes) [inline baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist/model.py:58]: Why did it add a space here? I think the comment should be split into a single-line first sentence and a possibly longer second sentence.

Comment (tanertopal) [reply baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist/model.py]: @danieljanes I don't know, but given this is a deprecated code, I'd suggest skipping to put too much effort into it. I made a slight adjustment. I would assume that @jafermarq will have a look anyway when migrating the code into the new baseline format.

Comment (jafermarq) [reply baselines/flwr_baselines/flwr_baselines/publications/leaf/femnist/model.py]: yes

**Review by danieljanes — APPROVED** [review]

**Pull request merged**